### DISCUSSION TITLE
Ventura erase-install Question

### DISCUSSION THREAD

**Discussion by thdenton** [grahampugh/erase-install#361] [Q&A]
We have been using erase-install for sometime now.

I have load of ventura M1 which I want to downgrade to Monterey would I be right in saying you script wont allow me to do that. This is my current command. I'm getting error around cant downgrade from current verison.

/Library/Management/erase-install/erase-install.sh --erase --version=12.6.3 --move --current-user --depnotify --update --max-password-attempts= infinite -nobrowse 

using the latest 29 version.

Thanks

**Comment by grahampugh** [comment]
Downgrade is not possible. See item 1 of the FAQs in the wiki.
